tests/stub/routing/test_routing_v4x3.py

```diff
@@ -672,7 +672,7 @@ def test_should_fail_when_writing_on_writer_that_returns_database_unavailable(se
                 **self.get_vars(),
                 "#FAILURE#":
                     '{"code": '
-                    '"Neo.ClientError.General.DatabaseUnavailable", '
+                    '"Neo.TransientError.Database.DatabaseUnavailable", '
```

Comment:
Why did this error code change?

Reply:
Because that's the real error code the server issues. I could and possibly should put this into a separate PR. I just dislike the overhead of PRs.